### PR TITLE
meteor: 1.2.0.1 -> 1.4.2.3

### PR DESCRIPTION
**Pull request by sorpaas**
###### Motivation for this change

Update Meteor version.

###### Things done

- [ ] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/servers/meteor/default.nix [hunk 1]
```diff
@@ -62,7 +62,7 @@ stdenv.mkDerivation rec {
     for p in $(find $out/packages -name '*.node'); do
       patchelf \
         --set-rpath "$(patchelf --print-rpath $p):${stdenv.cc.cc.lib}/lib" \
-        $p
+        $p || true
```
Comment (matthiasbeyer) [inline pkgs/servers/meteor/default.nix]: Why this change?

Comment (sorpaas) [inline pkgs/servers/meteor/default.nix]: Several binaries in the new meteor tar are not patch-able. I add this so those failed binaries get ignored and the build passes.

**Comment by grahamc** [comment]
Applied in 4a80a97ca9, thank you!
